export function* throwError(err) {
  console.log('throwError', err);
}
